src/home-assistant.html

```diff
@@ -50,70 +51,70 @@
   }
 };
 
-Polymer({
-  is: 'home-assistant',
-
-  hostAttributes: {
-    icons: null,
```

Comment:
I couldn't find anywhere this was used. Is this a dead attribute? I only found references to the Polymer property.

Reply:
From what I understand, host attributes will be read and set as values on our properties. We then use it to check when icons are done loading.

Reply:
Oh I guess this is only necessary to get data set as host attribute, this is not needed to get the attribute read. Guess we can indeed remove this